### ISSUE TITLE
Problem installing dependencies: which "linter"?

### ISSUE THREAD

**Issue by TerryCavanagh** [snowkit/atom-haxe#65]
Here's what happens right now when you try to install the required package dependencies for this package:

![linter](https://cloud.githubusercontent.com/assets/2874150/20480984/be5080ec-afdc-11e6-9952-919afc9e3eef.png)

Same situation with `autocomplete-plus`.

Not sure if this is something you care about, but it completely gates beginners from using the plugin!

**Comment by jeremyfa** [comment]
Right, to make it smoother, it would be possible to make the plugin install dependencies automatically (which is done by some other plugins). A good compromise, to prevent from being too intrusive, could be to prompt, once, the user if he wants to install "linter" etc... (autocomplete-plus is already included in atom by default).

**Comment by TerryCavanagh** [comment]
Automatic dependency installation sounds great to me, if it's possible!

**Comment by ruby0x1** [comment]
@jeremyfa is there a readme outdated that mentions autocomplete-plus to install? We should revise that! 

Also, we could add a linter checkbox which installs the package manually if there's an API endpoint for that, otherwise we could just install the dependency anyway and the checkbox just toggles it's usage by default if necessary.

**Comment by jeremyfa** [comment]
Yes, installing it automatically and allowing to disable its usage sounds fine to me.
